### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by Ade179**
### form validation by @Jaymelfah  and @Ade179 

- correct GitHub workflow was implemented
- no linters error
- form contact validation on client  works fine

### CODE REVIEW

File: script/index.js [hunk 1]
```diff
@@ -252,4 +295,31 @@ cardsArray.forEach((list) => {
...
+  return false;
+}
+
+// stop button default action
+const form = document.querySelector('form');
+form.addEventListener('submit', (event) => {
+  if (valid() === false) {
+    event.preventDefault();
+  }
+});
```
Comment (cynthiainga) [inline script/index.js:321]: - [ ] As in the project requirement, if a user submits the contact form with the email field containing uppercase characters an error message should show up but that is not the case for you. When I submit a contact form with the email field containing uppercase characters it submits correctly, please kindly fix it 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Ade179, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script/index.js (hunk 1 above)
Comment (Ade179) [reply script/index.js:321]: ok, i will do so now

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Ade179,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**